packages/store-api/src/platforms/vtex/utils/facets.ts

```diff
@@ -0,0 +1,21 @@
+export interface SelectedFacet {
+  key: string
+  value: string
+}
+
+/**
+ * Transform facets from the store to VTEX platform facets.
+ * For instance, the channel in Store becames trade-policy in VTEX's realm
+ * */
+export const transformSelectedFacet = ({ key, value }: SelectedFacet) => {
+  switch (key) {
+    case 'channel':
+      return { key: 'trade-policy', value }
+
+    case 'slug':
+      return { key: 'id', value: value.split('-').reverse()[0] }
```

Comment:
Maybe we should pop the array to avoid reversing it

```suggestion
      // gets last element after split
      return { key: 'id', value: value.split('-').pop() }
```

Reply:
Nice IDK about this pop on js. :satisfaction: